lib/docs/scrapers/phoenix.rb

```diff
@@ -47,6 +46,14 @@ class Phoenix < UrlScraper
       end
     }
 
+    version '1.3' do
+      self.release = '1.3.4'
+    end
+
+    version '1.4' do
+      self.release = '1.4.10'
+    end
```

Comment:
Why do we need two separate releases for a minor update? I don't see any major breaking changes between 1.3.4 and 1.4.10, so I think keeping it to one version is a better idea.

Reply:
Sorry, I think I misread the release notes. There is indeed no breaking changes to the docs.